doc/UsersGuide/source/solving.rst

```diff
@@ -73,6 +73,35 @@ solver and furthermore it has the following IDA specific flags:
 :ref:`idaNonLinConvCoef <simflag-idaNonLinConvCoef>`,
 :ref:`idaMaxErrorTestFails <simflag-idaMaxErrorTestFails>`.
 
+
+.. _sundials_cvode :
+
+CVODE
+~~~~~
+
+The CVODE solver is part of sundials: SUite of Nonlinear and
+DIfferential/ALgebraic equation Solvers :cite:`Hindmarsh:2005`.
+CVODE solves initial value problems for ordinary differential equation (ODE)
+systems with variable-order, variable-step multistep methods.
+
+In OpenModelica, CVODE uses a combination of Backward Differentiation
+Formulas (varying order 1 to 5) as linear multi-step method and a modified
+Newton iteration with fixed Jacobian as non-linear solver per default.
+This setting is advised for stiff problems which a very common for Modelica
```

Comment:
Shouldn't it be `are` instead of `a` here.

`This setting is advised for stiff problems which are very common for Modelica models.`

Reply:
Yes